analyzer/src/main/kotlin/managers/Sbt.kt

```diff
@@ -52,8 +52,9 @@ class Sbt(
         private val PROJECT_REGEX = Regex("\\[info] \t [ *] (.+)")
         private val POM_REGEX = Regex("\\[info] Wrote (.+\\.pom)")
 
-        // Batch mode (which suppresses interactive prompts) is only supported on non-Windows, see
-        // https://github.com/sbt/sbt-launcher-package/blob/d251388/src/universal/bin/sbt#L86.
+        // Batch mode (which suppresses interactive prompts) is not supported on Windows, see
```

Comment:
You speak about it not being supported on Windows, but then list the Linux link first. How about swapping the order of the links, and saying "compare" instead of "see"?

Reply:
Done, also added a few more commits.